cli/dcoscli/job/main.py

```diff
@@ -239,16 +238,12 @@ def _kill(job_id, run_id, all=False):
 
 
 def _list(json_flag=False):
-    """
+    """ Provides a list of jobs along with their history summary
```

Comment:
Nit: it actually loads the (comprehensive) `history`, not only the `historySummary`. `historySummary` provides merely the `lastSuccessAt/LastFailureAt` timestamps, while `history` also provides a list of all job runs and their statuses (capped at 1k runs iirc).

If I'm right, loading `historySummary` would suffice for the matter but would require changing `last_success = _str_to_datetime(job['history']['lastSuccessAt'])` to `last_success = _str_to_datetime(job['historySummary']['lastSuccessAt'])`, same for `lastFailureAt`.